pkg/common/yaml/yaml.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package yaml
+
+import (
+	"io/ioutil"
+	"path"
+)
+
+import (
+	perrors "github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+// LoadYMLConfig Load yml config byte from file
+func LoadYMLConfig(confProFile string) ([]byte, error) {
+	if len(confProFile) == 0 {
+		return nil, perrors.Errorf("configure file name is nil")
+	}
+
+	if path.Ext(confProFile) != ".yml" {
+		return nil, perrors.Errorf("configure file name{%v} suffix must be .yml", confProFile)
+	}
+
+	return ioutil.ReadFile(confProFile)
+}
+
+// UnmarshalYMLConfig Load yml config byte from file, then unmarshal to object
+func UnmarshalYMLConfig(confProFile string, out interface{}) ([]byte, error) {
+	confFileStream, err := LoadYMLConfig(confProFile)
+	if err != nil {
+		return confFileStream, perrors.Errorf("ioutil.ReadFile(file:%s) = error:%v", confProFile, perrors.WithStack(err))
+	}
+	return confFileStream, yaml.Unmarshal(confFileStream, out)
+}
+
```

Comment:
1.this func may obey ‘Single Responsibility Principle’
2. is there need in returning byte content?

Reply:
I simply copy the code from dubbo-go. in dubbo-go they want to store the raw data to the config entity. we can just remove this return since we don't have this need.

Reply:
done